### PR TITLE
:sparkles: Add simple shadow ban

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

File: resources/views/profile.blade.php [hunk 1]
```diff
@@ -42,10 +42,16 @@
                 @if(!$user->isAuthUserBlocked)
                     <span class="fs-2">
                         <span class="font-weight-bold"><i class="fa fa-route d-inline"></i>&nbsp;{{ number($user->train_distance / 1000) }}</span><span
-                                class="small font-weight-lighter">km</span>
+                            class="small font-weight-lighter">km</span>
                         <span class="font-weight-bold ps-sm-2"><i class="fa fa-stopwatch d-inline"></i>&nbsp;{!! durationToSpan(secondsToDuration($user->train_duration * 60)) !!}</span>
-                        <span class="font-weight-bold ps-sm-2"><i class="fa fa-dice-d20 d-inline"></i>&nbsp;{{ $user->points }}</span><span
-                            class="small font-weight-lighter">{{__('profile.points-abbr')}}</span>
+                        @if(!$user->shadow_banned)
```
Comment (xanderio) [inline resources/views/profile.blade.php]: Shadow banned users should only disappear from the dashboard, there profile shouldn't be affected by this.

**Review by xanderio — CHANGES_REQUESTED** [review]

**Comment by MrKrisKrisu** [comment]
I'll add an interaction in admin panel in another pr.

**Review by xanderio — APPROVED** [review]

**Comment by jeyemwey** [comment]
Do we also want to ban the people from the global dashboard and the enroute views? Maybe thats something for another PR later :)

**Comment by MrKrisKrisu** [comment]
> Do we also want to ban the people from the global dashboard and the enroute views? Maybe thats something for another PR later :)

Sounds useful.

**Review by jeyemwey — APPROVED** [review]

**Pull request merged**